soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -125,11 +276,14 @@ def map_phys_mem():
         tot = 1 + int(runx("awk '/HugePages_Total/ {print $2}' /proc/meminfo"))
         os.system(f"echo {tot} > /proc/sys/vm/nr_hugepages")
 
-    hugef = open(HUGEPAGE_FILE, "w+")
+    hugef_name = HUGEPAGE_FILE + str(stream_id)
+    hugef = open(hugef_name, "w+")
```

Comment:
Probably not a big deal, but note that these pages are MUCH larger than we need, so it's a little wasteful to have one per stream.  Longer term it might be worth trying to optimize usage.  But then... a few dozen MB isn't that much on a modern system.

Reply:
I considered writing a quick'n'dirty allocator on top of a single huge page for just that reason... but thought there were better uses of time, for the moment.